### DISCUSSION TITLE
请问DatePicker如何返回格式化后的值？

### DISCUSSION THREAD

**Discussion by acccccccb** [DouyinFE/semi-design#2135] [Q&A]
```javascript
<Form.DatePicker
      label={"证件有效期-开始日期"}
      field="certificate_validity_date_start"
      type={"date"}
      format={"yyyy-MM-dd"}
      placeholder="开始日期"
      disabledDate={(date) => {
          const today = dayjs().startOf("day");
          return dayjs(date).isAfter(today, "day");
      }}
      onChange={() => {}}
  />```

certificate_validity_date_start 最终值为：`2024-03-18T16:00:00.000Z`
怎么才能更改为`yyyy-MM-dd`格式？
我已经尝试在onChange中改变了格式，但提交时仍然为：`2024-03-18T16:00:00.000Z`

**Comment by pointhalo** [comment]
Datepicker 的 onChange 默认第一个入参是 Date 对象，第二个入参是 date string
![image](https://github.com/DouyinFE/semi-design/assets/88709023/e8a3c70a-fe6d-498b-a194-ef8f46380d9e)

Form.Datepicker 默认监听的是onChange的第一个入参，如果你是想存储在 formState中的是 date string，有两种方法
- onChangeWithDateFirst 设为false
- 或者 通过 Field 的 convert API 做一次转换 <Form.Datepicker   field='xxx'    convert={(val)  => format(val, 'yyyy-MM-dd') } />
![image](https://github.com/DouyinFE/semi-design/assets/88709023/94c7d0f7-46a1-4776-b133-f336f474791a)

**Reply by acccccccb** [reply]
问题已解决，谢谢！
